io.github.ar_.apple-flinger.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: io.github.ar_.apple-flinger
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: run.sh
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+    - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: apple-flinger
+    buildsystem: simple
+    build-commands:
+      - install -D apple-flinger.jar -t ${FLATPAK_DEST}/
+      - install -Dm644 logo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 ${FLATPAK_ID}.desktop "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - install -Dm755 run.sh /app/bin/run.sh
+    sources:
+    
+      - type: file
+        path: io.github.ar_.apple-flinger.desktop
+
+      - type: file
+        url: https://gitlab.com/ar-/apple-flinger/-/raw/master/art/logo/logo.png?ref_type=heads&inline=false
```

Comment:
1. never from a branch
2. why is this a different repo than the .jar. And doesn't match the app-id either.